ddcz/models/used/users.py

```diff
@@ -85,70 +91,78 @@ class UserProfile(models.Model):
-    popis_uzivatele = MisencodedCharField(max_length=255, blank=True, null=True)
-    nova_posta = models.IntegerField(default=0)
-    skin = MisencodedCharField(max_length=10)
-    reputace = models.IntegerField(default=0)
-    reputace_rozdel = models.PositiveIntegerField(default=0)
-    status = MisencodedCharField(max_length=1)
-    reg_schval_datum = models.DateTimeField(blank=True, null=True, auto_now_add=True)
-    indexhodnotitele = models.DecimalField(
-        max_digits=4, decimal_places=2, default=-99.99
+    pii_display_permissions = MisencodedCharField(
+        max_length=15, db_column="vypsat_udaje"
+    )
+    icon = MisencodedCharField(
+        max_length=25, blank=True, null=True, db_column="ikonka_uzivatele"
+    )
+    description_raw = MisencodedCharField(
+        max_length=255, blank=True, null=True, db_column="popis_uzivatele"
+    )
+    new_post_no = models.IntegerField(default=0, db_column="nova_posta")
+    skin = MisencodedCharField(max_length=10, db_column="skin")
+    reputation = models.IntegerField(default=0, db_column="reputace")
+    reputation_available = models.PositiveIntegerField(
+        default=0, db_column="reputace_rozdel"
     )
-    reload = MisencodedCharField(max_length=1)
-    max_level = models.IntegerField(blank=True, null=True)
+    status = MisencodedCharField(max_length=1, db_column="status")
+    registration_approved_date = models.DateTimeField(
+        blank=True, null=True, auto_now_add=True, db_column="reg_schval_datum"
+    )
+    evaluator_index = models.DecimalField(
+        max_digits=4, decimal_places=2, default=-99.99, db_column="indexhodnotitele"
+    )
+    reload = MisencodedCharField(max_length=1, db_column="reload")
+    max_level = models.IntegerField(blank=True, null=True, db_column="max_level")
     api_key = MisencodedCharField(unique=True, max_length=40, blank=True, null=True)
     tavern_bookmarks = models.ManyToManyField(
-        "TavernTable", through="TavernBookmark", through_fields=("id_uz", "id_stolu")
+        "TavernTable",
+        through="TavernBookmark",
+        through_fields=("user_profile", "tavern_table"),
     )
 
     class Meta:
         db_table = "uzivatele"
 
     @property
     def slug(self):
-        slug = create_slug(self.nick_uzivatele)
+        slug = create_slug(self.nick)
         if not slug:
             slug = "neznamy"
             # TODO: log an error
         return slug
 
     @property
     def icon_url(self):
-        if not self.ikonka_uzivatele:
+        if not self.icon:
             return None
         else:
-            return urljoin(settings.USER_ICON_MEDIA_ROOT_URL, self.ikonka_uzivatele)
+            return urljoin(settings.USER_ICON_MEDIA_ROOT_URL, self.icon)
 
     @property
     def is_author(self):
         return self.author is not None
 
     @property
     def is_female(self):
-        return self.pohlavi_uzivatele in ["Žena", "®ena"]
+        return self.gender in ["Žena", "®ena"]
```

Comment:
Just an idea - since this is really a gender and currently there are people who are non-binary (that I still don't fully understand how it works), should we do something like non-binary option and then have this function returning string "" or "a" or "o"?

Reply:
Well yes, once we get rid of the old version. 

Although I am tempted to say that I'd keep `salutation` and just delete this one since we don't care.